_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/scripts/@rhd/js/namespace.js

```diff
@@ -19,21 +19,6 @@ app.downloadManagerBaseUrl = "#{site.download_manager_base_url}";
 */
 app.cache = {};
 
-
-/*
-  JS templates
-*/
-app.templates = {};
-app.templates.searchpageTemplate = '#{partial "searchpage.html.slim"}';
-app.templates.miniBuzzTemplate = '#{partial "mini_buzz_template.html.slim"}';
-app.templates.productBuzzTemplate = '#{partial "product_buzz_template.html.slim"}';
-app.templates.buzzTemplate = '#{partial "buzz_template.html.slim"}';
-app.templates.termsAndConditionsTemplate = '#{partial "terms_and_conditions.html.slim"}';
-app.templates.bookTemplate = '#{partial "book.html.slim"}';
-app.templates.connectorTemplate = '#{partial "product-connectors-item.html.slim"}';
-
-app.templates.productStackoverflowTemplate = '#{partial "product_stackoverflow_template.html.slim"}';
-app.templates.stackoverflowTemplate = '#{partial "stackoverflow_template.html.slim"}';
```

Comment:
I don't think this file needs to stay anymore. It's all in drupal-namespace.js I think.